Makefile

```diff
@@ -23,7 +23,7 @@ start-api:
 		echo "$(VINYLDNS_REPO) not found in your GOPATH (necessary for acceptance tests), getting..."; \
 		git clone https://$(VINYLDNS_REPO) $(GOPATH)/src/$(VINYLDNS_REPO); \
 	fi
-	$(GOPATH)/src/$(VINYLDNS_REPO)/bin/docker-up-vinyldns.sh
+	$(GOPATH)/src/$(VINYLDNS_REPO)/bin/docker-up-vinyldns.sh --timeout 120
```

Comment:
If this is pulling the most recent script, using `-a`/`--api-only` should do the trick since we don't need the portal.